doc/guides/west/manifest.rst

```diff
@@ -156,82 +115,105 @@ bases are respectively ``https://example.com/base1`` and
+         url: https://github.com/user/project-three
          revision: abcde413a111
 
-This example has three projects:
+In this manifest:
 
 - ``proj1`` has remote ``remote1``, so its Git fetch URL is
-  ``https://example.com/base1/proj1`` (note that west adds the ``/`` between
-  the URL base and project name). This project will be cloned at path
-  ``extra/project-1`` relative to the west installation's root directory.
-  Since the project has no ``revision``, the current tip of the ``master``
-  branch will be checked out as a detached ``HEAD``.
-
-- ``proj2`` has the same remote, so its fetch URL is
-  ``https://example.com/base1/proj2``. Since the project has no ``path``
-  specified, it will be cloned at ``proj2`` (i.e. a project's ``name`` is used
-  as its default ``path``). The commit pointed to by the ``v1.3`` tag will be
-  checked out.
-
-- ``proj3`` has fetch URL ``https://example.com/base2/proj3`` and will be
-  cloned at path ``proj3``. Commit ``abcde413a111`` will be checked out.
-
-Each element in the ``projects`` sequence can contain the following keys. Some
-of the description refers to the ``defaults`` subsection, which will be
-described next.
-
-- ``name``: Mandatory, the name of the project. The fetch URL is formed as
-  remote url-base + '/' + ``name``. The name cannot be one of the reserved
-  values "west" and "manifest".
-- ``remote``: The name of the project's remote. If not given, the ``remote``
-  value in the ``defaults`` subsection is tried next. If both are missing, the
+  ``https://example.com/base1/proj1``. The remote ``url-base`` plus a ``/`` is
+  prepended to the project ``name`` to form the URL in this case.
+
+  Locally, this project will be cloned at path ``extra/project-1`` relative to
+  the west installation's root directory, since it has an explicit ``path``
+  attribute with this value.
+
+  Since the project has no ``revision`` specified, ``master`` is used by
+  default. The current tip of this branch will be fetched and checked out as a
+  detached ``HEAD`` when west next updates this project.
+
+- ``proj2`` has a ``remote`` and a ``repo-path``, so its fetch URL is
+  ``https://example.com/base2/my-path``. The ``repo-path`` attribute, if
+  present, overrides the default ``name`` when forming the fetch URL.
+
+  Since the project has no ``path`` attribute, its ``name`` is used by
+  default. It will be cloned into a directory named ``proj2``. The commit
+  pointed to by the ``v1.3`` tag will be checked out when west updates the
+  project.
+
+- ``proj3`` has an explicit ``url``, so it will be fetched from
+  ``https://github.com/user/project-three``.
+
+  Its local path defaults to its name, ``proj3``. Commit ``abcde413a111`` will
+  be checked out when it is next updated.
+
+The list of project keys and their usage follows. Sometimes we'll refer to the
+``defaults`` subsection; it will be described next.
+
+- ``name``: Mandatory, the name of the project. The name cannot be one of the
```

Comment:
You use the word ``Optional.`` below, so you should be consistent here by writing ``Mandatory.  The name of the project.``